NuGet/Realm.Database/Realm.Database.nuspec

```diff
@@ -35,6 +35,11 @@
         <dependency id="System.Runtime.InteropServices" version="4.3.0"/>
         <dependency id="System.Runtime.InteropServices.RuntimeInformation" version="4.3.0"/>
       </group>
+      <group targetFramework="uap">
```

Comment:
Do we have to explicitly state we depend on the .NET standard library or is it assumed?